i18n/it.toml

```diff
@@ -178,6 +178,9 @@ other = "Prima di cominciare"
 [ui_search_placeholder]
 other = "Cerca"
 
+[subscribe_button]
```

Comment:
Hi @Arhell .
What do you think about adding the `subscribe_button` field before the `ui_search_placeholder` field?
The fields are listed alphabetically.

Reply:
Hi, done, thanks